src/Symfony/Component/Form/Extension/Core/Type/DateIntervalType.php

```diff
@@ -192,6 +195,21 @@ public function configureOptions(OptionsResolver $resolver)
             return array_fill_keys($timeParts, $placeholder);
         };
 
+        $labelsNormalizer = function (Options $options, array $labels) {
+            return array_replace(array(
+                'years' => null,
+                'months' => null,
+                'days' => null,
+                'weeks' => null,
+                'hours' => null,
+                'minutes' => null,
+                'seconds' => null,
+                'invert' => 'Negative interval',
+            ), array_filter($labels, function ($label) {
```

Comment:
No, because it'll filter any empty value.
It case someone simply want to disable the label, he'll set it to `false`. But `false` would be removed and thus impossible to disable the label rendering from options.